.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,28 @@
+name: RuboCop
+
+on:
+  push:
+    branches:
+      - main
+      - v1
+  pull_request:
+    branches:
+      - main
+      - v1
+
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.3
```

Comment:
You mean to define a repository environment variable like
 `RUBOCOP_RUBY_VERSION`?
I think it's easier for others to understand if you write the version directly here, so I prefer this way.

Reply:
Then why did you set `RUBY_VERSION: 3.3` in integration_tests.yml? I don't have a preference for a specific method, but I would like to maintain consistency across workflow files, IMO. As unit_test.yml uses a matrix for Ruby versions, I recommended to use `RUBY_VERSION` environment variable here.

Reply:
> Then why did you set `RUBY_VERSION: 3.3` in integration_tests.yml?

I think you're seeing an old version of integration_tests.yml. The current one doesn't have `RUBY_VERSION`: https://github.com/cookpad/arproxy/blob/4f5fc7c3efef5ddbaf7b0dd14a951315d8295e90/.github/workflows/integration_tests.yml

Anyway, since I don't have a strong opinion about this, I added `RUBY_VERSION` to rubocop.yml: 8553fbb